test/libsolidity/semanticTests/events/simple.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+    event E();
+}
+
+contract Test {
+    function f() public {
+        emit C.E();
```

Comment:
Could you make this a bit more complicated by adding another event called `E` to `Test`, but which has a different signature? Just to check that the correct event is emitted and that the test infrastructure can also cope with that.